docs/src/formula.md

```diff
@@ -55,7 +62,7 @@ It's often convenient to include main effects and interactions for a number of
 variables.  The `*` operator does this, expanding in the following way:
```

Comment:
Can some more words be added here to explain what `*` is ?

Reply:
It isn't even entirely clear to me (as someone with out an R background) what `&` is saying

Reply:
the example kind of makes it clear what `*` does, but at least there must be some words to describe it.
It probanly should be a seperate paragraph to `&`